packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -146,6 +144,28 @@ class DiffPaneController extends DisposableController {
       ),
     );
   }
+
+  ClassFilterButton _classFilterButton() => ClassFilterButton(
+        filter: core.classFilter,
+        onChanged: applyFilter,
+        rootPackage: serviceManager.rootInfoNow().package,
+      );
```

Comment:
this is a bit of a strange pattern. We shouldn't have controller classes be responsible for creating widgets. Instead, pass a `diffPaneController` parameter to ClassFilterButton

Reply:
replaced with controller, thanks